packages/atomic/src/components/commerce/atomic-commerce-interface/atomic-commerce-interface.new.stories.tsx

```diff
@@ -10,27 +11,66 @@ async function initializeCommerceInterface(canvasElement: HTMLElement) {
   );
   await commerceInterface!.initialize(getSampleCommerceEngineConfiguration());
 }
+
+const {events, args, argTypes, template} = getStorybookHelpers(
+  'atomic-commerce-interface',
+  {excludeCategories: ['methods']}
+);
+
 const meta: Meta = {
   component: 'atomic-commerce-interface',
   title: 'Commerce/Interface',
   id: 'atomic-commerce-interface',
-  render: renderComponent,
-  parameters,
-  play: async (context) => {
+  render: (args) => template(args),
+  parameters: {
+    ...parameters,
+    actions: {
+      handles: events,
+    },
+  },
+  decorators: [(story) => html`<div id="code-root">${story()}</div>`],
+
+  afterEach: async (context) => {
     await initializeCommerceInterface(context.canvasElement);
     const searchInterface = context.canvasElement.querySelector(
       'atomic-commerce-interface'
     );
     await searchInterface!.executeFirstRequest();
   },
   argTypes: {
-    'attributes-language': {
-      name: 'language',
-      type: 'string',
+    ...argTypes,
+    engine: {
+      ...argTypes,
+      control: {
+        disable: true,
+      },
+      table: {
+        defaultValue: {summary: undefined},
+      },
+    },
+    urlManager: {
+      ...argTypes.urlManager,
+      control: {
+        disable: true,
+      },
+      table: {
+        defaultValue: {summary: undefined},
+      },
+    },
+    i18n: {
+      ...argTypes.i18n,
+      control: {
+        disable: true,
+      },
+      table: {
+        defaultValue: {summary: undefined},
+      },
```

Comment:
It make sense for engine etc to be properly documented from the JSDOC and all, but we do not want user tinkering with them using the control panel in storybook.